api/README.md

```diff
@@ -16,9 +16,10 @@ You can install the project dependencies with the following command from within
 Once the dependencies have been installed (above) you can run the example server by running the following commands:
 
 > pipenv shell
+>
 > python app.py
```

Comment:
I think it's better to format this way

```bash
pipenv shell
python app.py
```

which you can achieve by enclosing the text in triple backticks (hard to print when this comment is rendered as well). Follow the initial backticks by `bash` for syntax highlighting 😎.

Reply:
Like here: https://raw.githubusercontent.com/maasglobal/sustainable-mobility-api/master/estimator/README.md

Reply:
K, changes made.